### ISSUE TITLE
Unable to access Profile Setting to unsubscribe mail list

### ISSUE THREAD

**Issue by SajanRajah** [SajanRajah/Capstone#1]
UpdateProfile.php was modified to allow for unsubscribing from the bulk mailing list. After modification, I am unable to access the Profile Setting even when logged in as Admin.

**Comment by Geethatnt** [comment]
Is this bug still not resolved?can i help you to resolve?

**Comment by SajanRajah** [comment]
Yes, thank you. Please review updateprofile.php in this repository.

**Comment by Geethatnt** [comment]
Hi Sajan can you please add the session_start() in the updateprofile.php? Check now it will work .
